### PR TITLE
Font Awesome

### PR DESCRIPTION
**Pull request by FiveEightyEight**
Font Awesome addition to index.html for future use in app.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -7917,11 +7953,13 @@
...
               "version": "1.0.2",
-              "bundled": true
+              "bundled": true,
+              "optional": true
             },
             "yallist": {
               "version": "3.0.3",
-              "bundled": true
+              "bundled": true,
+              "optional": true
```
Comment (JoshuaKGoldberg) [inline package-lock.json:7962]: Quick tip: it looks like some of you folks have different `npm` versions. You can use an `engines` property in your `package.json` to help stop these weird unrelated changes: https://docs.npmjs.com/files/package.json#engines

Comment (FiveEightyEight) [reply package-lock.json:7962]: Interesting, we've been wondering why this kept happening. Not sure what node version everyone is using at the moment. Would allowing  >= 8v be ok for now?

**Review by pamelaabreu — APPROVED** [review]
